src/PhpSpec/CodeGenerator/Generator/Argument/StringBuilder.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PhpSpec\CodeGenerator\Generator\Argument;
+
+class StringBuilder
+{
+    /**
+     * @param \ReflectionParameter[] $parameters
+     *
+     * @return string
+     */
+    public function buildFromReflectionParameters(array $parameters, string $targetClassNamespace) : string
+    {
+        return implode(', ', array_map(function (\ReflectionParameter $parameter) use ($targetClassNamespace) {
+            return $this->buildArgument($parameter, $targetClassNamespace);
+        }, $parameters));
+    }
+
+    private function buildArgument(\ReflectionParameter $parameter, string $targetClassNamespace) : string
+    {
+        $parameterName = '$' . $parameter->getName();
```

Comment:
Even though this is trivial at the moment it might be better to extract this to a method, to make the level of abstraction in this method consistent